tests/helpers/Participant.ts

```diff
@@ -383,4 +404,87 @@ export class Participant {
 
         return await localDisplayName.getText();
     }
+
+    /**
+     * Gets avatar SRC attribute for the one displayed on local video thumbnail.
+     */
+    async getLocalVideoAvatar() {
+        const avatar
+            = this.driver.$('//span[@id="localVideoContainer"]//img[contains(@class,"userAvatar")]');
+
+        return await avatar.isExisting() ? await avatar.getAttribute('src') : null;
```

Comment:
```suggestion
        return await (avatar.isExisting() ? avatar.getAttribute('src') : null);
```

Reply:
I think this would work.

Reply:
Nope.

<img width="933" alt="image" src="https://github.com/user-attachments/assets/048d49d3-0225-4a96-aeb8-bb94b19c0758" />